packages/fxa-content-server/app/scripts/models/reliers/oauth.js

```diff
@@ -317,16 +318,16 @@ var OAuthRelier = Relier.extend({
     return !!(
       this._config &&
       this._config.scopedKeysEnabled &&
-      this.has('keysJwk')
+      this.has('keysJwk') &&
+      this._wantsScopeThatHasKeys
```

Comment:
This extra check prevents us from allocating a `keyFetchToken` if the relier doesn't actually want any keys.